ssr-opengraph/src/types.ts

```diff
@@ -15,11 +15,23 @@ export interface NFT {
   currentOwner: string;
   issuer: string;
   meta?: NFTMeta;
-  price: string;
+  price?: string;
 }
 
 export interface NFTEntity {
   data: {
     item: NFT;
   };
 }
+
+export interface CollectionEntity {
+  data: {
+    collection: NFT;
+  };
+}
+
+export interface ListEntity {
+  data: {
+    items: NFT[];
+  };
+}
```

Comment:
can we write some shared type library ? 

I don't think that copypasting is good solution imo for long-term

Reply:
https://github.com/kodadot/packages/issues/41